### PR TITLE
Sysdig team for sso

### PR DESCRIPTION
**Pull request by patricksimonian**
team infrastructure code

### CODE REVIEW

File: openshift/sysdig-team/team.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: ops.gov.bc.ca/v1alpha1
+kind: SysdigTeam
+metadata:
+  name: devops-sso-prod-sysdigteam
+spec:
+  team: 
+    description: The Sysdig Team for the SSO Prod Namespaces
+    users:
+    - name: ConradBoydElliottGustafson
```
Comment (ShellyXueHan) [inline openshift/sysdig-team/team.yaml:9]: it's not github account but email address

Comment (patricksimonian) [reply openshift/sysdig-team/team.yaml:9]: oh but which email address? their github id's registered email addres?

File: openshift/sysdig-team/team.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+kind: SysdigTeam
+metadata:
+  name: devops-sso-prod-sysdigteam
+spec:
+  team: 
+    description: The Sysdig Team for the SSO Prod Namespaces
+    users:
+    - name: c.gustafson@cgi.com
+      role: ROLE_TEAM_EDIT 
+    - name: patrick.simonian@gov.bc.ca
```
Comment (patricksimonian) [inline openshift/sysdig-team/team.yaml:11]:
```suggestion
    - name: patricksimonian@gmail.com
```

File: docs/SYSDIG.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## Sysdig
+
+SSO can utilize Sysdig to benefit daily operation and monitoring of the dev/test/prod services. Access to sysdig is managed through CRD's. To create a sysdig team run `oc apply -f team.yaml` found in `openshift/sysdig` in your __tools__ namespace. 
+
```
Comment (patricksimonian) [inline docs/SYSDIG.md:4]:
```suggestion
detailed instructions can be found [on the devhub](https://developer.gov.bc.ca/OpenShift-User-Guide-to-Creating-and-Using-a-Sysdig-Team-for-Monitoring)
```

File: docs/SYSDIG.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+1. go to https://app.sysdigcloud.com/#/explore/overview/l:3600
+
+2. Login via Openid 
+
+![sysdig login](./images/sysdig-login.png)
+
+3. Enter in the correct openid alias
+![sysdig openid](./images/sysdig-openid.png)
+
+4. From there you can navigate to your dashboards or create new ones. You will need to bind permissions to the sysdig team name `6d70e7-team` to grant access.
```
Comment (patricksimonian) [inline docs/SYSDIG.md:16]:
```suggestion

```

**Review by nvunnamm — APPROVED** [review]

**Pull request merged**